config/gnbsim.yaml

```diff
@@ -11,6 +11,10 @@ info:
 configuration:
   singleInterface: false #default value
   execInParallel: false #run all profiles in parallel
+  httpServer: # Serves APIs to create/control profiles on the go
```

Comment:
add GSLab copyright.. Latest copyright should be at the top.

Reply:
Done